hadoop-hdfs-project/hadoop-hdfs-rbf/src/main/java/org/apache/hadoop/hdfs/server/federation/fairness/AbstractRouterRpcFairnessPolicyController.java

```diff
@@ -75,4 +77,17 @@ protected void insertNameServiceWithPermits(String nsId, int maxPermits) {
   protected int getAvailablePermits(String nsId) {
     return this.permits.get(nsId).availablePermits();
   }
+
+  @Override
+  public String getAvailableHandlerOnPerNs() {
+    JSONObject json = new JSONObject();
+    for (Map.Entry<String, Semaphore> entry : permits.entrySet()) {
+      try {
+        json.put(entry.getKey(), entry.getValue().availablePermits());
```

Comment:
Let's extract entry.getKey() and entry.getValue() to have a particular name.

Reply:
Updated.